examples/scripts/read_example_dataset.py

```diff
@@ -1,11 +1,12 @@
+import os
 from typing import Optional
 
-import pkg_resources
 from pyspark.sql import DataFrame
 
 from spark_expectations.core import get_spark_session
 
 spark = get_spark_session()
+CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
 
 
 def spark_expectations_read_employee_dataset(prepend: Optional[str] = None) -> DataFrame:
```

Comment:
some example notebooks for consumers are using this methods to load sample data (.csv) as proof of concept in demo notebooks 
- https://engineering.nike.com/spark-expectations/latest/delta/#example-write-to-delta

Would this be packaged with the pypy artifact or do we even still want to continue including those files or we will just update documentation to pass in inline sample data as we are doing with notebooks in this repository